02-express-tutorial/app.js

```diff
@@ -1,12 +1,30 @@
 console.log('Express Tutorial');
 const express = require('express');
-const {products} = require('./data');
+const { products, people } = require('./data');
+const peopleRouter = require('./routes/people');
+const cookieParser= require('cookie-parser');
+const authMiddleware = require('./middleware/auth');
 const app = express();
 
-app.use(express.static("./public"));
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+app.use(cookieParser());
+app.use(express.static('./methods-public'));
+app.use("/api/v1/people", peopleRouter);
+
+
+const logger = (req, res, next) => {
+    console.log(`[${new Date().toISOString()}] ${req.method} ${req.url}`);
+    next();
+};
+app.use(logger);
+/*app.get('/', logger, (req, res) => {
```

Comment:
Avoid leaving commented out code in your files. You can always check previous commits if you need to remember how it looked before, so you can safely remove it.